bin/photos.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+'use strict'
+
+var easyimage = require('easyimage')
+var fs = require('fs')
+
+fs.readdir('./images/gallery', function (err, items) {
```

Comment:
@simonmcmanus 
Yes, that's correct.
The `photo.js` is not yet automated, hence it is still pointing to images/gallery to build the thumbnails. No integration with json yet.

The reason is simple -  I haven't had the time to implement this yet. :blush: 
I was planning to do this on a separate PR in the next couple days.

Basically, we currently have a gallery that works but is not automated.